lib/sqlalchemy/util/_concurrency_py3k.py

```diff
@@ -14,9 +20,26 @@
     # Issue for context: https://github.com/python-greenlet/greenlet/issues/173
 
     class _AsyncIoGreenlet(greenlet.greenlet):
-        def __init__(self, fn, driver):
-            greenlet.greenlet.__init__(self, fn, driver)
-            self.driver = driver
+        if copy_context is None or greenlet.__version__ < "0.4.17":
```

Comment:
If we could enforce the required version of greenlet to be 1.0 or above, we could remove the branches here and keep the last branch only.